### DISCUSSION TITLE
How to Host Bentley Digital Twin (iModel) on a Custom or Free Domain?

### DISCUSSION THREAD

**Discussion by Zubair-KPM** [iTwin/community#57] [Q&A]
I'm currently working on a project using Bentley Digital Twin technology and hosting an iModel, and I'd like to host my application on either a custom domain or a free domain. Could someone guide me through the process from start to finish?

I have tried the below process using netlify.

Reference: https://medium.com/itwinjs/deploying-the-itwin-viewer-to-a-web-host-d45c5cfdf0cf#d22e

Can anyone tell me how to do so?

**Comment by aruniverse** [comment]
Hi! Just to confirm, were you able to successfully deploy your viewer to netlify, and just need to assign a custom domain? Or did you run into issues deploying?
